WebUI/src/components/SettingsBasic.vue

```diff
@@ -61,14 +65,34 @@ const themeToDisplayName = (theme: Theme) => {
     }
 }
 
-
 const modelSettings = reactive<KVObject>(Object.assign({}, toRaw(globalSetup.modelSettings)));
+
+const modelSettingsWatcher = watchEffect(() => {
+  modelSettings.llm = globalSetup.modelSettings.llm;
+  modelSettings.sd_model = globalSetup.modelSettings.sd_model;
+  modelSettings.lora = globalSetup.modelSettings.lora;
+});
+
+onMounted(() => {
+  cancelModelSettingsChange();
```

Comment:
I am also failing to understand, why we need to cancel onMounted and why we would need to activate the watcher. Just from dashing through docs I would assume the way the watcher is used looks like "faulty" to me